lib/routes/assets.js

```diff
@@ -14,6 +14,38 @@ var comment = require('commenting'),
 mime.default_type = 'text/javascript';
 
 module.exports = function (app) {
+  const spec = app.spec;
+
+  /*
+   * API for serving latest files
+   */
+  app.routes.get('/assets/-/:pkg/:env/:version?', function (req, res) {
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
+
+    spec.findBuild(
+      spec.fromRequest(req),
+      function (err, build) {
+        if (err) { return next(err); }
+
+        //
+        // TODO (indexzero): proper sanitization on this untrusted
+        // user input.
+        //
+        const { filter } = req.query;
+        if (filter) {
+          res.send(
+            build.files.filter(file => {
+              return path.extname(file) === filter;
```

Comment:
should we limit this to just extension? if we are we should support `.js` and `js` so with or without the `.`. My thought is that we could make it string includes based but idk if thats too complicated

Reply:
Having `filter` be the extension name where the leading `.` is stripped automatically was my intention.